src/install.js

```diff
@@ -276,124 +280,120 @@ Run ppm -v after installing Git to see what version has been detected.\
         }
       }
 
-      return this.requestPackage(packageName, (error, pack) => {
-        if (error != null) {
-          this.logFailure();
-          return callback(error);
-        } else {
-          if (packageVersion == null) { packageVersion = this.getLatestCompatibleVersion(pack); }
-          if (!packageVersion) {
-            this.logFailure();
-            callback(`No available version compatible with the installed Atom version: ${this.installedAtomVersion}`);
-            return;
+      const commands = [];
+      try {
+        const pack = await this.requestPackage(packageName);
+        packageVersion ??= this.getLatestCompatibleVersion(pack);
+        if (!packageVersion) {
+          throw `No available version compatible with the installed Atom version: ${this.installedAtomVersion}`;
```

Comment:
```suggestion
          throw `No available version compatible with the installed Pulsar version: ${this.installedAtomVersion}`;
```

Again not your error or fault, just a good idea to fix this while seeing it